app/controllers/api/v1/books_controller.rb

```diff
@@ -4,4 +4,10 @@ def show
     book_id = params[:id]
     render json: BookSerializer.new(BooksFacade.get_book(book_id))
   end
+
+  def search
+    searched_title = params[:title]
+    render json: BookSerializer.new(BooksFacade.get_searched_books(searched_title))
```

Comment:
We could call this "index" since I think it is technically our index function, however, search may be necessary depending on what other methods we have for retrieving a list of books..?